db/model/generator.js

```diff
@@ -79,7 +79,7 @@ module.exports = function generator(seq, dataTypes) {
       type: dataTypes.ARRAY(dataTypes.STRING(constants.fieldlen.normalName)),
       allowNull: false,
     },
-    keywords: {
+    tags: {
```

Comment:
I think you have to change the column name in the migration otherwise it won't happen from the sync cuz the table already exists